src/Element.php

```diff
@@ -514,4 +522,13 @@ public function getMessages()
     {
         return $this->messages;
     }
+    
+    /**
+     * 
+     * @return boolean
+     */
+    public function hasValue()
```

Comment:
This must be tested with a unit test at least

Reply:
Ok, I've to write a test. I'll check an existing test and make one. Thanks for the suggestion.